src/comm/TcpAcceptor.cc

```diff
@@ -250,17 +249,16 @@ void
 Comm::TcpAcceptor::logAcceptError(const ConnectionPointer &tcpClient) const
 {
     AccessLogEntry::Pointer al = new AccessLogEntry;
-    CodeContext::Reset(al);
     al->tcpClient = tcpClient;
     al->url = "error:accept-client-connection";
     al->setVirginUrlForMissingRequest(al->url);
     ACLFilledChecklist ch(nullptr, nullptr, nullptr);
```

Comment:
I believe this checklist creation code belongs to the code context of the transaction being logged (i.e. `al`). The checklist is created specifically for logging that transaction and is not used by the transaction-neutral TcpAcceptor service code. Please move it inside the CallBack() enclosure.